cli/tests/testdata/run/workspaces/basic/deno.json

```diff
@@ -0,0 +1,9 @@
+{
+  "workspaces": [
+    "foo",
+    "bar"
```

Comment:
It would be nice to add a few more error tests. For example:

1. A member that's specified like `"../other_folder"`
2. A member that's a member of another member: `"./member1", "./member1/folder"`.